src/control/lib/control/system.go

```diff
@@ -1246,3 +1242,74 @@ func SystemGetProp(ctx context.Context, rpcClient UnaryInvoker, req *SystemGetPr
 
 	return resp, nil
 }
+
+// SystemRebuildManageReq sends an interactive pool rebuild request to each pool in a DAOS system.
+type SystemRebuildManageReq struct {
+	unaryRequest
+	msRequest
+	OpCode PoolRebuildOpCode // Which pool rebuild operation to trigger
+	Force  bool              // Applies to stop interactive rebuild only
+}
+
+// PoolRebuildManageResult describe the result of an interactive rebuild operation on a given pool.
+type PoolRebuildManageResult struct {
+	ID      string            `json:"id"`
+	OpCode  PoolRebuildOpCode `json:"op_code"`
```

Comment:
No change required, just asking - is the PoolRebuildOpCode needed in the per-pool responses? Since the requester will know what operation was asked for?